SDK Showcase/SDK Showcase/Views/ContentView.swift

```diff
@@ -57,7 +56,8 @@ struct ContentView: View {
                     
                     Section(content: {
                         TextResult(result: system.isSDKInitialized ? "✅ SDK initialized" : "❌ SDK not initialized \(errorValue)")
-                        TextResult(result: system.registationState == .registered ? "👤 User registered as \(userData.userId ?? "")" : "🚫 User not registered")
+                        TextResult(result: registrationStatus)
+                        TextResult(result: authenticationStatus)
```

Comment:
While you start registering the new user and the "at least one user is registered" result is displayed, the result is changed to "no registered" <- which is wrong

Reply:
done.

Reply:
just checked -> tapping browser registration changes the status to "no registered"

Reply:
Yeah - I'm struggling with that from yesterday. So the problem is setting and getting `shouldShowBrowser` that is in the System but the system doesn't know about registered users that is potentially available "above" in AppState. Any ideas?

Reply:
I will fix this by holding previous state.